.github/workflows/python-app.yml

```diff
@@ -31,28 +31,36 @@
        with:
          virtualenvs-create: true
          virtualenvs-in-project: true
-     - name: Cache Poetry virtualenv
-       id: cache
-       uses: actions/cache@v2
-       with:
-         path: .venv
-         key: venv-${{ runner.os }}-#{{ hashFiles('**/poetry.lock') }}
+     # - name: Cache Poetry virtualenv
+     #   id: cache
+     #   uses: actions/cache@v2
+     #   with:
+     #     path: .venv
+     #     key: venv-${{ runner.os }}-#{{ hashFiles('**/poetry.lock') }}
+     - name: install pip
+       run: poetry run python -m pip install --upgrade pip
+     - name: Hacked install of pybind11
+       run: |
+         poetry run python -m pip install pybind11
+     - name: Hacked install of fasttext
+       run: |
+         poetry run python -m pip install fasttext
      - name: Install dependencies
        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
        run: poetry install --no-interaction --no-root
-     - name: Install 
+     - name: Install
        run: poetry install --no-interaction
      - name: Test with pytest
        working-directory: ./tests
        run: |
-         source ../.venv/bin/activate       
+         source ../.venv/bin/activate
          pytest --cov --cov-fail-under=64
      - name: flake8 linter
        run:  |
          source ./.venv/bin/activate
          python -m flake8 --count --max-complexity 10 --ignore E501,W503 --show-source --statistics --exclude ./.venv .
      - name: Run mypy
        run: |
-         source ./.venv/bin/activate       
+         source ./.venv/bin/activate
          pip install mypy
          mypy --install-types --non-interactive --namespace-packages
```

Comment:
Something is fishy here -- these are my changes from a different pull request that has merged.  It leads me to believe this PR is missing changes from current `develop`

Reply:
Given the force push that occurred, you might need to cut a new branch and cherry pick the other commits. This shouldn’t be showing something that has already merged to develop. If this gets merged as is I suspect weird things will happen. You could try doing a merge of latest develop if the rebase doesn’t seem to be working.